### ISSUE TITLE
Openssl version changed in OSX High Sierra

### ISSUE THREAD

**Issue by moodh** [hbldh/pybankid#10]
OSX High sierra seems to have changed openssl implementations to LibreSSL which causes a check in pybankid to crash.

(flow) moodh:~/pybankid$ which openssl
/usr/bin/openssl

(flow) moodh:~/pybankid$ openssl version
LibreSSL 2.2.7

Here's the relevant code:
https://github.com/hbldh/pybankid/blob/master/bankid/certutils.py#L76
```
        p = subprocess.Popen(['openssl', 'version'], stdout=subprocess.PIPE)
        sout, serr = p.communicate()
        if not sout.decode().lower().startswith('openssl'):
            raise NotImplementedError(
                "OpenSSL executable could not be found. "
                "Splitting cannot be performed.")
        print(sout.strip())
        openssl_executable = 'openssl'
```

I guess both has to be supported.
I can fix a PR if you'd like but since the change is trivial you might just want to fix it yourself. :)

Thanks!

**Comment by hbldh** [comment]
Interesting; will get this done shortly.
The `certutils` has not really been a prioritized part of PyBankID, more like a convenience tool for splitting certs that hopefully works for most people. The check that fails is pretty hacky to begin with, so it is not surprising that it fails.

**Comment by hbldh** [comment]
@moodh Just to make sure; you have tried changing to `startswith('libressl')` and made sure that it works? I have no Mac to test it with myself...

**Comment by moodh** [comment]
No, I added the homebrew path before i ran the script :) I'll try adapting the real code and let you know!

**Comment by hbldh** [comment]
Clone the repo and use the latest commit on the develop branch; implemented a fix there.

**Comment by moodh** [comment]
Just tested it and the `split_certificate` function works.
Thanks!

**Comment by hbldh** [comment]
Fixed in cfb6afc25544b2acd3f488c2914634d7d361acfa. Included in version 0.6.1 release on PyPI.
